### ISSUE TITLE
Open Intro question error

### ISSUE THREAD

**Issue by wgw** [swirldev/swirl_courses#210]
The recorded answer is incorrect, so I can't skip the question (see below). 

In general, the lesson should indicate question ids so the user can refer to the question by id when errors occur (or better yet, have a commenting /issue flagging integrated into the lesson so errors can be flagged directly from swirl).  

```
  |=======================================================         |  86%

| According to the bar graph, what is the average price
| for a 'large' car (in $1000s)?

> 25000
[1] 25000

| You almost had it, but not quite. Try again.

| Look at the highest value on the y-axis that the
| \large\" bar reaches."

> 50000
[1] 50000

| Give it another try.

| Look at the highest value on the y-axis that the
| \large\" bar reaches."

> skip()

| Entering the following correct answer for you...

> 22- 26

| Not exactly. Give it another go.

| Look at the highest value on the y-axis that the
| \large\" bar reaches."
```

**Comment by seankross** [comment]
Did you try a number between 22 and 26 as an answer?

**Comment by wgw** [comment]
Yes,  I gave 22, though I was tempted to give 22-26. 

Sorry, now I see that I misunderstood that skipping means that I still have to answer (Doh!). I thought the machine was giving itself the answer.  (First time I used "skip".)

That's me being obtuse. 

So the swirl "issue" is closed (I will open an issue on my obtuseness with my creator, though I do not have his/her/its email...). 

Thanks for the quick reply.
